### DISCUSSION TITLE
Interpretation of BioAnalyzer results?

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1433] [Q&A] [answered]
Alright, I extracted DNA from 3 Tanner crab samples from 2005 and then ran them on the Qubit and BioAnalyzer. Goal was to assess DNA quality from these archived samples and determine if a large-scale project is feasible.

I'm not entirely sure how to interpret these BioAnalyzer results - any feedback on what this looks like in terms of quality?

Links to: 

[Qubit results](https://github.com/afcoyle/historical_hemat/blob/main/analyses/DNA_extraction/test_01_2005_bairdi/qubit_results.csv)

[Notes on extraction](https://afcoyle.github.io/2022-03-16-DNA_extraction_results/)

And BioAnalyzer results are below. Note: The 4th sample ran on the BioAnalyzer was a blank meant to test whether the gel (expired) was still good.

![Electropherogram](https://raw.githubusercontent.com/afcoyle/historical_hemat/main/analyses/DNA_extraction/test_01_2005_bairdi/Electropherogram.PNG)

![Gel](https://raw.githubusercontent.com/afcoyle/historical_hemat/main/analyses/DNA_extraction/test_01_2005_bairdi/Gel.png)

**Comment by kubu4** [comment, marked as answer]
Looks like the markers weren't run or didn't work. Will need to be re-run.

I wouldn't use expired reagents for Bioanalyzer.

Expired chips are okay, though.

**Reply by aspencoyle** [reply]
Gotcha, how can you tell the marker didn't work? Trying to learn all this! And do you know if we have fresh reagents (for DNA specifically)?

**Reply by kubu4** [reply]
I can tell the markers didn't work because they're not present in any of the lanes. The marker serves two purposes:

- positive control
- time reference

There should be a small marker and a large marker in all lanes. These are represented as tall narrow peaks _very_ early (small marker) in the measurement time and _very_ late (large marker) in the measurement time.

I'd doubt we have any usable Bionalyzer reagents. Expiration date is usually about 6 months after receipt, and is indicated on the box.

**Reply by kubu4** [reply]
I'll order a new set of reagents Monday.

**Reply by aspencoyle** [reply]
Thanks a bunch, Sam! Can you let me know when they get in?

**Reply by kubu4** [reply]
@afcoyle Kit is scheduled for delivery today. FedEx tracking number: 271310528357
